cmd/commands/migrate.go

```diff
@@ -115,16 +138,19 @@ func runHelmMigrate(ctx context.Context, opts *MigrateOptions) error {
 		return err
 	}
 
+	opts.runtimeNamespace = *runtime.Metadata.Namespace
```

Comment:
i decided to just put some values on the opts, and send them down into the funcs, instead of adding more and more parameters